src/middleware/healthRoutes/index.js

```diff
@@ -18,8 +18,8 @@ const status = {
 const getReadyStatus = async ({ dependencies, logger }) => {
   const dependenciesPromises = await dependencies.map(async (dependency) => {
     try {
-      await getData({ endpoint: dependency.endpoint, logger });
-      logger.info(`${dependency.name} is healthy`);
+      await getData({ endpoint: dependency.endpoint, logger, showLogs: false });
+      logger.debug(`${dependency.name} is healthy`);
```

Comment:
I don't think we need to change this as this is when we call the endpoint on another service

Reply:
So if we don't change this. We will constantly see this logged also.

![](https://files.slack.com/files-pri/T0KNVDB9Q-F013URNPFRB/image.png)

Reply:
why are we calling it so many times? surely that is the issue here?

Reply:
That's the way the platform works from my understanding. It constantly calls the health check endpoints to see if the app is still running as expected. So it can synchronise correctly. I might be wrong.

Reply:
ah ok so when they call our endpoint we then call the api endpoint(s)